src/escpos/escpos.py

```diff
@@ -711,13 +712,14 @@ def control(self, ctl, pos=4):
         elif ctl.upper() == "CR":
             self._raw(CTL_CR)
         elif ctl.upper() == "HT":
-            if not (1 <= pos <= 16):
+            if not (0 <= count <= 32 and 1 <= tab_size <= 255 and count * tab_size < 256):
```

Comment:
What about separating each part in a line using no extra variables?
if not (0 <= count <= 32 and
          1 <= tab_size <= 255 and
          count * tab_size < 256):